### DISCUSSION TITLE
boot from usb drive/stick (apple m1, m2)

### DISCUSSION THREAD

**Discussion by nan0tEch** [utmapp/UTM#4867] [Ideas]
It's a reiterative question, it would be welcome to create a disposable vm where it could boot from my usb stick where pfsense/ truenas or raspbian is on installed for debugging. 

The idea is a few times asked on the iOS version, but this could be a killer feature on the Mac imho.

**Comment by jhass** [comment]
Meanwhile you can:

1. create a new VM
2. delete the disk drive in the settings
3. start it
4. enable the USB passthrough for your USB stick
5. reset the VM and spam ESC immediately
6. optionally change the boot order in boot management
7. or just boot from the USB stick via the boot options

It's a little fiddly but it works.

**Reply by mx1up** [reply]
I enabled the USB passthrough and reset, but the usb drive does not show up in the boot order or boot options.. 
It is a Samsung PSSD T7 with linux on installed
